packages/requisitions/src/SupplierRequisition/DetailView/DetailView.tsx

```diff
@@ -10,7 +10,7 @@ import {
 } from '@openmsupply-client/common';
 import { useRequestRequisition, useIsRequestRequisitionDisabled } from '../api';
 import { Toolbar } from './Toolbar';
-import { Footer } from './Footer';
+import { Footer } from './Footer/Footer';
```

Comment:
line 29 - the disabled state on AppBarButtons is reversed

Reply:
Ah thanks, oops! Lucky it doesn't do anything yet 😁